tests/conftest.py

```diff
@@ -48,6 +47,8 @@
 from tornado.platform.asyncio import AnyThreadEventLoopPolicy
 import sys
 import pkg_resources
+from asyncio.futures import wrap_future
+from asyncio.events import get_event_loop
```

Comment:
do we prefer this style or do we consistently go for 

```python
import ayncio

asyncio.get_event_loop()
```
or even

```python

from asyncio import get_event_loop

```

Reply:
The first. This is one of the flake8 checks that we disabled, only module level imports. Cannot find the reference.

Reply:
Fix to asyncio.get_event_loop()